test/modules/TestAnalyzer/AggregateCloudOperationReport.cs

```diff
@@ -49,5 +49,21 @@ public class CloudOperationReport
 
             public DateTime LastReceivedAt { get; set; }
         }
+
+        bool IsPassedHelper()
```

Comment:
why name it like this?

Reply:
The IsPassed property needs more complex logic than what we had previously ([Expression Body Definition](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-operator#expression-body-definition)) to determine whether it is passed. The purpose of this helper is to provide a place where we can provide a multi-line implementation.

Seeing as how these fields' purposes are the same and the only difference was needing a mechanism to provide multi-line implementation, this name seemed ok. Do you have another suggestion?

Reply:
I agreed to use a method to wrap the logic.  but why just name this method "IsPassed"?  Do you worry about other dev will move it back to inline/expression body if property and method name are the same?

Reply:
It can't have the same name as the property

Reply:
Sorry. my bads.